store/tikv/region_request.go

```diff
@@ -167,6 +169,14 @@ func (s *RegionRequestSender) sendReqToRegion(bo *Backoffer, ctx *RPCContext, re
 	if e := tikvrpc.SetContext(req, ctx.Meta, ctx.Peer); e != nil {
 		return nil, false, errors.Trace(e)
 	}
+	// judge the store limit switch.
+	limit := storeutil.StoreLimit.Load()
+	if limit > 0 {
```

Comment:
Prefer `if limit := storeutil.StoreLimit.Load(); limit > 0` to avoid pollute external scope.

Reply:
Addressed.